third_party/3/pyspark/serializers.pyi

```diff
@@ -117,5 +119,10 @@ class ChunkedStream:
     @property
     def closed(self): ...
 
+def write_with_length(obj: Any, stream: Any): ...
+def pack_long(value): ...
 def read_int(stream): ...
+def read_long(stream): ...
+def read_bool(stream): ...
 def write_int(value, stream): ...
+def write_long(write_long, stream): ...
```

Comment:
The name of the variable doesn't look right here?

Reply:
Good catch, thanks! Probably some copy-paste :)